tools/clean/clean.sh

```
@@ -1,2 +1,5 @@
+echo "Stopping server..."
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)